front/src/modules/study/utils.ts

```diff
@@ -12,7 +12,6 @@ export const createSelectOptions = (
   const { t } = useTranslation('operationalStudies/study');
   if (isEmpty(list)) return [{ label: t('nothingSelected').toString() }];
   return [
```

Comment:
You can remove the first [], it is not useful anymore
```suggestion
  return sortBy(
    list.map((key) => ({ id: key, label: t(`${translationList}.${key}`) })),
    'value'
  );
```

Reply:
Indeed, thanks. Done in https://github.com/osrd-project/osrd/compare/34a9cd65605ae07add316f5cb14cc4da9fb159c3..4212613e8a4011a0740f5698950bfc84d37f5aba.